### DISCUSSION TITLE
Argument for citing code

### DISCUSSION THREAD

**Discussion by samm82** [samm82/TestingTesting#120] [Show and tell]
If anyone ever needs a tangible example of why it's important to cite your code, I realized that the functionality provided by the following package wasn't present in the preamble for my paper. By following the link from the comment, I was able to quickly verify that it _was_ the package I was looking for! 😁 It would have taken some Googling if I hadn't have cited it or had asked ChatGPT how to do it (yes, you can link to ChatGPT threads, but if you're asking AI what packages to use, you're probably not citing them either).

https://github.com/samm82/TestGen-Thesis/blob/587453cfc9cc136869c52c15fa5b003720c44396/preamble.tex#L184-L185

**Comment by smiths** [comment]
Thanks for sharing @samm82.  Glad the story has a happy ending.  :smile:
